<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>623</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="SendGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>基于DLT/645-2007多功能电表通信协议</string>
    </property>
    <widget class="QLineEdit" name="Phase1AddressLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
    </widget>
    <widget class="QLabel" name="SinglePAdr">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>21</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>单相电表地址A5-A0</string>
     </property>
    </widget>
    <widget class="QLabel" name="ThreePAddr">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>65</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>三相电表地址A5-A0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Phase3AddressLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>84</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>[FE FE FE FE 68H A0 A1 A2 A3 A4 A5 68H C L DATA CS 16H]：帧格式</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Frame_1">
     <property name="geometry">
      <rect>
       <x>313</x>
       <y>41</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Frame_2">
     <property name="geometry">
      <rect>
       <x>313</x>
       <y>85</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="DataTypeEdit_2">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>83</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0201FF00</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="DataTypeEdit_1">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>39</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>02010100</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="DataType_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>64</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据标识DI3-DI0</string>
     </property>
    </widget>
    <widget class="QLabel" name="DataType">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据标识DI3-DI0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ResultBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>791</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>识别结果</string>
    </property>
    <widget class="QTextBrowser" name="Result_textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>731</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Msg_Box">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>791</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>消息</string>
    </property>
    <widget class="QTextBrowser" name="MsgTextArea_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>501</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="MsgTextArea_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>261</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>待定相电压</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A相电压</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>10</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B相电压</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>10</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>C相电压</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>16</width>
       <height>221</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>16</width>
       <height>221</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>30</y>
       <width>16</width>
       <height>221</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SettinggroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>791</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QPushButton" name="StartButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EndButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>结束</string>
     </property>
    </widget>
    <widget class="QLabel" name="TimeSynchronized_Label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>S(秒)</string>
     </property>
    </widget>
    <widget class="QLabel" name="TimeSynchronized_Label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>h(小时)</string>
     </property>
    </widget>
    <widget class="QLabel" name="CollectFrequency_Label">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>11</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>发送间隔</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CollectFrequency_Button">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>11</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>60</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>120</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="CollectTimeButton">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>51</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="CollectTime_Label">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>51</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>采集时间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TimeSynchronized_Button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>时间同步</string>
     </property>
    </widget>
    <widget class="QLabel" name="TimeSynchronized_Label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>由于同时请求电压，可不进行同步</string>
     </property>
    </widget>
    <zorder>CollectFrequency_Label</zorder>
    <zorder>CollectFrequency_Button</zorder>
    <zorder>CollectTimeButton</zorder>
    <zorder>CollectTime_Label</zorder>
    <zorder>TimeSynchronized_Button</zorder>
    <zorder>TimeSynchronized_Label</zorder>
    <zorder>StartButton</zorder>
    <zorder>EndButton</zorder>
    <zorder>TimeSynchronized_Label_2</zorder>
    <zorder>TimeSynchronized_Label_3</zorder>
   </widget>
   <widget class="QLabel" name="MainWindowComStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>串口未开启！</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>460</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;发帧数：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>460</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;收帧数：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="SendCount_Text">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>460</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="RecvCount_Text">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>460</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aa00;&quot;&gt;开始时间：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="StartTimeLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>461</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>593</x>
      <y>460</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aa00;&quot;&gt;结束时间：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="EndTimeLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>460</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空消息区</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_System">
    <property name="title">
     <string>系统(Y)</string>
    </property>
    <addaction name="QuitBar"/>
   </widget>
   <widget class="QMenu" name="menu_Connect">
    <property name="title">
     <string>通讯</string>
    </property>
    <addaction name="CommunicateBar"/>
    <addaction name="DisconnectBar"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="AboutBar"/>
   </widget>
   <addaction name="menu_System"/>
   <addaction name="menu_Connect"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="QuitBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="CommunicateBar">
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="DisconnectBar">
   <property name="text">
    <string>断开</string>
   </property>
  </action>
  <action name="AboutBar">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
